t/01-wasm/002-shm_queue_sanity.t

```diff
@@ -25,3 +26,20 @@ __DATA__
 --- no_error_log
 [error]
 [crit]
+
+
+
+=== TEST 2: queue - with minimum size, macOS (arm64)
```

Comment:
I think that instead of duplicating the test, we should have the size value be conditional based on the OS/arch with `main_config eval` blocks. We do this several times already in the tests (and in recent changes in main as well).